javascript/sdk/lib/controllers/iam.ts

```diff
@@ -17,12 +17,12 @@ export default class IAMController {
   }
 
   async newAccount(
-    { email, name }: CreateAccountParams,
+    { email, name, company }: CreateAccountParams,
```

Comment:
should I default company to `""` @makk94  or will BE take care of this and return is as an empty string if no value is passed?